src/constants/localization.ts

```diff
@@ -95,8 +95,8 @@ export type TooltipStrings = {
     stringParams?: any;
     urlConfigs?: LinksConfigs;
   }) => {
-    title: string;
-    body: string;
+    title?: string;
+    body?: string;
```

Comment:
`body` should be required. I think that a tooltip without a `body` defeats the purpose of a tooltip